src/Pelagos/Bundle/AppBundle/Resources/public/css/search.css

```diff
@@ -121,3 +121,9 @@ h1{
     display:inline-block;
 }
 
+#resgrp-facet{
```

Comment:
a class name would be better, then you can apply it to all facets?

Reply:
I'm not sure if other facets will have really long lists. I think we wouldn't want to have scrolls for all facets